hack/lib/test.sh

```diff
@@ -394,16 +396,16 @@ kube::test::version::diff_assert() {
         echo "${bold}${red}"
         echo "FAIL! ${diff_msg}"
         echo "  Expected: "
-        echo "$(cat ${original})"
+        cat "${original}"
```

Comment:
Fix for this:

```
In ./hack/lib/test.sh line 397:
        echo "$(cat ${original})"
             ^------------------^ SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
                    ^---------^ SC2086: Double quote to prevent globbing and word splitting.
```